daemon/containerd/image_pull.go

```diff
@@ -206,10 +206,6 @@ func (i *ImageService) pullTag(ctx context.Context, ref reference.Named, platfor
 	infoHandler := snapshotters.AppendInfoHandlerWrapper(ref.String())
 	opts = append(opts, containerd.WithImageHandlerWrapper(infoHandler))
 
-	// Allow pulling application/vnd.docker.distribution.manifest.v1+prettyjws images
-	// by converting them to OCI manifests.
-	opts = append(opts, containerd.WithSchema1Conversion) //nolint:staticcheck // Ignore SA1019: containerd.WithSchema1Conversion is deprecated: use Schema 2 or OCI images.
-
```

Comment:
Heh, just stumbled on this one Yesterday and was planning to open a PR for that; wondering if we should split up things; also if we could split up the dependency updates in some smaller commits if we want to review those changes, because it's pretty huge.